service/matching/tasklist/matcher.go

```diff
@@ -117,9 +116,8 @@ func newTaskMatcher(config *config.TaskListConfig, fwdr *Forwarder, scope metric
 //   - task is matched and consumer returns error in response channel
 func (tm *TaskMatcher) Offer(ctx context.Context, task *InternalTask) (bool, error) {
 	var err error
```

Comment:
this `err` variable is confusing and doesn't need to be defined at this scope. I'd get rid of it to avoid future bugs

Reply:
ah, yep - apparently I just looked past it too many times.  will do 👍